src/components/DocumentStatus/DocumentStatus.tsx

```diff
@@ -0,0 +1,153 @@
+      location: string;
+    }[];
+  }[];
+  className?: string;
+}
+
+export const IssuedBy = ({ verificationStatus }: DocumentStatusProps) => {
+  const dnsFragmentName = "OpenAttestationDnsTxt";
+  const dnsFragment = verificationStatus.find((status) => status.name === dnsFragmentName);
+  const dnsIdentity = dnsFragment?.data?.every((issuer: { status: string }) => issuer.status === "VALID");
+
+  return (
+    <h3 id="issuedby" className={`pl-2 mb-0 issuedby`}>
+      <span className="mr-1">Issued by</span>
+      <span className="domain">
+        {dnsIdentity
+          ? `${dnsFragment?.data.map((issuer: { location: string }) => issuer.location.toUpperCase()).join(" and ")}`
+          : "Unknown"}
+      </span>
+    </h3>
+  );
+};
+
+export const DocumentStatus = ({ verificationStatus, className }: DocumentStatusProps) => {
+  const positiveFragments = getAllButRevokeFragment(verificationStatus);
+  const negativeFragments = [getRevokeFragment(verificationStatus)];
+  const hashValid = isValid(positiveFragments, ["DOCUMENT_INTEGRITY"]);
+  const issuedValid = isValid(positiveFragments, ["DOCUMENT_STATUS"]) && !certificateNotIssued(positiveFragments);
+  const revokedValid = isValid(negativeFragments, ["DOCUMENT_STATUS"]);
+  const identityValid = isValid(verificationStatus, ["ISSUER_IDENTITY"]);
+
+  return (
+    <section className={`bg-blue-lighter py-3 ${className}`}>
+      <div className="container-custom">
+        <div id="document-status" className="statusbar">
+          <div className="container-fluid">
+            <div className="row align-items-center">
+              <div className="col-12 col-xl-4 mb-3 mb-xl-0">
+                <div className="row align-items-center no-gutters">
+                  <div className="col-auto">
+                    <SvgIcon>
+                      <SvgIconCheckCircle />
+                    </SvgIcon>
+                  </div>
+                  <div className="col">
+                    <IssuedBy verificationStatus={verificationStatus} />
+                  </div>
+                </div>
+              </div>
+              <div className="col-12 col-lg-3 col-xl-2 mb-2 mb-lg-0">
+                <StatusCheck valid={hashValid} messageSet={MESSAGES[TYPES.HASH]} />
+              </div>
+              <div className="col-12 col-lg-3 col-xl-2 mb-2 mb-lg-0">
+                <StatusCheck valid={issuedValid} messageSet={MESSAGES[TYPES.ISSUED]} />
+              </div>
+              <div className="col-12 col-lg-3 col-xl-2 mb-2 mb-lg-0">
+                <StatusCheck valid={revokedValid} messageSet={MESSAGES[TYPES.REVOKED]} />
+              </div>
+              <div className="col-12 col-lg-3 col-xl-2 mb-2 mb-lg-0">
+                <StatusCheck valid={identityValid} messageSet={MESSAGES[TYPES.IDENTITY]} />
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+  );
+};
+
+export const DocumentStatusDefault = styled(DocumentStatus)`
```

Comment:
it's a bit weird to have a new component with the normal style (and a weird name)

I would suggest to put the style in the component or use the css prop.